tests/IntegrationTests/WebAppUiTests/TestingWebAppLocally.cs

```diff
@@ -49,46 +40,23 @@ public async Task ChallengeUser_MicrosoftIdentityFlow_LocalApp_ValidEmailPasswor
             await page.GotoAsync(UrlString);
             LabResponse labResponse = await LabUserHelper.GetDefaultUserAsync().ConfigureAwait(false);
 
-            try
-            {
-                // Act
-                Trace.WriteLine("Starting Playwright automation: web app sign-in & call Graph");
-                string email = labResponse.User.Upn;
-                await UiTestHelpers.PerformLogin_MicrosoftIdentityFlow_ValidEmailPasswordCreds(page, email, labResponse.User.GetOrFetchPassword());
+            // Act
+            Trace.WriteLine("Starting Playwright automation: web app sign-in & call Graph");
+            string email = labResponse.User.Upn;
+            await UiTestHelpers.FirstLogin_MicrosoftIdentityFlow_ValidEmailPassword(page, email, labResponse.User.GetOrFetchPassword());
 
-                // Assert
-                await Assertions.Expect(page.GetByText("Welcome")).ToBeVisibleAsync();
-                await Assertions.Expect(page.GetByText(email)).ToBeVisibleAsync();
-            }
-            catch (Exception ex)
-            {
-                Assert.Fail($"the UI automation failed: {ex}");
-            }
-            finally
-            {
-                p.Kill(true);
-            }
+            // Assert
+            await Assertions.Expect(page.GetByText("Welcome")).ToBeVisibleAsync();
+            await Assertions.Expect(page.GetByText(email)).ToBeVisibleAsync();
+        } catch (Exception ex)
```

Comment:
catch and finally should be on new lines.

Reply:
done